### DISCUSSION TITLE
Delete selected row doesn't remove the row from selectedflatrows.

### DISCUSSION THREAD

**Discussion by alverdal** [TanStack/table#2625] [General]
Hi,

When I delete a selected row from state the row itself is removed in the table but the selected rows is not updated and therefore the row after my deleted row is now selected. 
How can I remove the selected id from the list of selected rows?
I'm using "autoResetSelectedRows: false" since I need to keep my selected rows during sorting, paging  (deleting single rows) etc.

Code example:
https://codesandbox.io/s/dazzling-dubinsky-1uxrg?file=/src/App.js

**Comment by Xentox-Phil** [comment]
Did you find a fix to this? I have the same problem! :D

**Reply by alverdal** [reply]
Sorry, I can't remember if I solved it by a workaround or if I found another table-tool to use. I don't have source code access to this project anymore so I cannot verify.
